packages/grafana-toolkit/src/cli/tasks/plugin.utils.ts

```diff
@@ -0,0 +1,78 @@
+import { Task, TaskRunner } from './task';
+import { getPluginJson } from '../../config/utils/pluginValidation';
+import { GitHubRelease } from '../utils/githubRelease';
+import path = require('path');
+
+// @ts-ignore
+import execa = require('execa');
+
+const releaseNotes = () => {
+  return execLine(`awk \'BEGIN {FS="##"; RS=""} FNR==3 {print; exit}\' CHANGELOG.md`);
+};
+
+const checkoutBranch = async (branchName: string, options: string): Promise<string> => {
+  const currentBranch = await execLine(`git rev-parse --abbrev-ref HEAD`);
+  const createBranch =
+    (await execLine(`git branch -a | grep ${branchName} | grep -v remote || echo 'No release found'`)) === branchName
+      ? ''
+      : '-b';
+  if (currentBranch !== branchName) {
+    return `git checkout ${createBranch} ${branchName}`;
+  }
+  return '';
+};
+
+export interface GithuPublishOptions {
+  dryrun?: boolean;
+  verbose?: boolean;
+}
+
+const execLine = async (line: string): Promise<string> => {
+  const { stdout } = await execa.shell(line);
+  return stdout;
+};
```

Comment:
is this the same as the promisify stuff?

https://github.com/grafana/grafana/blob/cc638e81f4d24d147941dfaf5e5a47ea1ab41b75/packages/grafana-toolkit/src/cli/tasks/plugin.build.ts#L15

Reply:
maybe ask @stevenvachon how to best do this -- should not need a special wrapper

Reply:
@dprokop suggested using execa. All this is doing is wrapping it so I didn't have to change all of the calls in the module.